src/pages/index.tsx

```diff
@@ -16,6 +16,25 @@ export default function Home() {
       refetchOnWindowFocus: false,
     }
   );
+  const addDeckMutation = trpc.useMutation('add-deck');
+
+  const [decks, setDecks] = React.useState<Deck[]>(data?.decks || []);
```

Comment:
I'm still kinda hazy on trpc's useQuery and data vs. isLoading in effect. useState should declare an initial value, which I would guess is empty on first render? Does the "initial state" of the hook change when data loads in or is it something where we should run setDecks on load?